<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI::ChooseGlobalDatabaseDialog</class>
 <widget class="QDialog" name="GUI::ChooseGlobalDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global database settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="buddy">
          <cstring>leName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPath">
         <property name="text">
          <string>&amp;Path</string>
         </property>
         <property name="buddy">
          <cstring>lePath</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="leName">
         <property name="placeholderText">
          <string>Enter name here...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lePath">
           <property name="placeholderText">
            <string>Enter path here...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbSelectFile">
           <property name="toolTip">
            <string>Select file on disk</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbReject">
       <property name="toolTip">
        <string>Close the dialog without any changes</string>
       </property>
       <property name="text">
        <string>&amp;Reject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAccept">
       <property name="toolTip">
        <string>Accept path</string>
       </property>
       <property name="text">
        <string>&amp;Accept</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
